content/roads_from_notre.rst

```diff
@@ -0,0 +1,114 @@
+Roads from Notre-Dame
+#####################
+
+:date: 2015-12-27 23:20
+:tags: cartographie, paris
+:category: carto
+:author: Tristram Gräbener
+
+All roads leads to Rome and it makes great maps! The project `Roads to Rome <http://roadstorome.moovellab.com/>`_ made a small buzz
+with their gorgeous maps showing a superposition of many routes that lead to Rome.
+
+They well deserve their success. The representation looks like a human cardiovascular system, whose heart is Rome.
+
+I was somewhat jealous for not having that idea. Nor have I the aesthetic talent to make a beautiful representation, so I was slightly condescending:
+“Meh. It’s only 20 lines of code for a Dijkstra and some plotting”.
+
+Having some time in Perpignan with my family for a week, I tried to copy their work. Here is some kind of tutorial.
+
+Due to my small 4 years old netbook and a lot of laziness to optimize my code, the example will only cover France and not whole Europe.
```

Comment:
"Since my laptop is only (...)" (c'est pas à cause de ton laptop (quel qu'il soit), c'est à cause du fait qu'il soit)